### ISSUE TITLE
User Reported Sound Issue

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/waves-intro#21]
From a user, from 1-14-2021:
**Test device**
iPad
**Operating System**

**Browser**
iPad purchased APP
**Problem description** 
My students are on ipads and there is no sound on the sound portion of this sim.  Even WITH the play tone checked.
**Steps to reproduce**
We had covered that higher sound pitch = higher frequency (prior knowledge) so that worked for us, but had we introduced the topic with this phenomenon, it would NOT have worked successfully.  I am on a mac and it plays here fine
**Visuals**

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Waves Intro‬
URL: https://phet.colorado.edu/sims/html/waves-intro/latest/waves-intro_en.html
Version: 1.0.2 2020-02-25 08:14:53 UTC
Features missing: applicationcache, applicationcache, touch
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36
Language: en-US
Window: 1384x703
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

I suggested finding the ringer and turning it on as this usually solves this type of problem. User replied:
>We open the app.  We run "Water" just fine.  But when we move to "Sound"...nothing.
Without ANY adjustments on buttons, we can completely close (including swipe up and get it off of recent used) the app and restart it as if we've never been there before.  
THEN we can immediately go into "Sound" and it will work just fine--tone is wonderful.

And continued with clarification:
>1) this was definitely more than one kid issue.  I teach about 90 kids and this was the "normal" on our iPad today--meaning way more than half had issues.  I don't install our school used apps (another dept) but we purchased the app during the summer and they were installed before school started (i.e. already on iPad when kids picked them up).

>2) no, this didn't "fix" it. If we leave sound and go to water and back, it is tone dead and same with going to light and returning to sound, it is tone dead.

>I will attach pics from my iPad to show you what I see as the app and info.
![image](https://user-images.githubusercontent.com/41024075/104753277-ff5a4300-5714-11eb-9664-e0fa2c6d6d4d.png)

I was unable to reproduce this on the iPad 6th gen iPadOS 14.3 test device.

**Comment by samreid** [comment]
I tested this on iPad 7 running 14.3 and heard the "play tone" consistently in the sound screen.  I'm not sure how to investigate further.

**Comment by kathy-phet** [comment]
Do we know what iOS they are on? and what version iPad they have?

**Comment by KatieWoe** [comment]
I don't think we do. I asked but they only sent a picture of the device, which isn't very helpful. I can ask again with instructions if @samreid thinks it would be helpful.

**Comment by samreid** [comment]
Yes, please. That sounds like a good next step.

**Comment by KatieWoe** [comment]
Email sent. I will update here when I hear back

**Comment by KatieWoe** [comment]
>It does this on my teacher ipad:
software version 14.3
model name iPad (6th gen)
model number MR7F2LL/A

>It does this on a student ipad:
software version 14.3
model name iPad (7th gen)
model number MW742LL/A

**Comment by KatieWoe** [comment]
I'm not sure where to go from here, since that's pretty similar to the test device and I couldn't reproduce it myself.

**Comment by samreid** [comment]
Indeed, my testing configuration matches the student configuration.  I also downloaded a fresh copy of the PhET app, and it still worked correctly.  I'm not sure how to proceed--maybe check in with @jbphet since he knows more about sound?  Or check in with the app-developer, since it seems to be specific to the app?

**Comment by jbphet** [comment]
@KatieWoe - Could you ask them to try out Friction and Faraday's Law and let us know if sound works on those sims?  It would be nice to figure out whether this is a general issue or something that is specific to Waves Intro.  And please be sure to thank them for their time.

**Comment by KatieWoe** [comment]
Email sent.

**Comment by arouinfar** [comment]
I'm going to close this issue for a few reasons.
* We were never able to reproduce ourselves.
* The user doesn't appear to have responded to further questions.
* The user's OS (14.3) is now several generations old and no longer supported by Apple.
* No other users reported this bug.
